core-it-suite/src/test/resources/mng-5600/settings-template.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't understand the need for this file

Reply:
The original author does not respond, so try to figure out. The repository contains 201 XML file with the same name but with different content. If you are familiar with IT tests in Maven you could tell is it needed or not. I guess the author wanted to follow convention.